Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+
+RUN apt-get update
+RUN apt-get install -y libboost-all-dev
+RUN apt-get install -y make
+RUN apt-get install -y g++
+
+WORKDIR /opt/webserver
+COPY . /opt/webserver
+
+RUN make clean && make
+
+CMD tar -cf - webserver
```

Comment:
Right. It might be more clear in a different file. It'll also make it easier to add lines. This is works fine though.